src/core/client/admin/routes/Configure/sections/General/FeaturedByConfig.tsx

```diff
@@ -0,0 +1,48 @@
+import { Localized } from "@fluent/react/compat";
```

Comment:
Can you please organize the imports in this file?

Reply:
✅